Track.h

```diff
@@ -577,7 +580,23 @@ inline float getScoreCalc(const unsigned int seedtype,
   float score_ = 0.f;
   ////// V2 of candidate score (simplified score, after fix for counts of # missing hits):
   score_ = Config::validHitBonus_*nfoundhits - Config::missingHitPenalty_*nmisshits - chi2;
-  if(pt<0.9f && seedtype==2) score_ -= 0.5f*(Config::validHitBonus_)*nfoundhits;
+  if(seedtype==2) {
+    score_ -= 0.5f*(Config::validHitBonus_)*nfoundhits;
+  }
+  if (seedtype==2 || seedtype==3) {
+    if (nfoundhits<=8) {
+      score_ -= 0.06f*(Config::validHitBonus_)*nfoundhits;
+    } else if (nfoundhits>12) {
+      score_ += 0.08f*(Config::validHitBonus_)*nfoundhits;
```

Comment:
I'm surprised these relatively small changes to the hit bonus for short and long tracks make a difference